app/_includes/snippets/gateway-operator/install_with_helm.md

```diff
@@ -12,10 +12,33 @@ helm repo update kong
 
 Install {{ site.kgo_product_name }} with Helm:
 
+{% if include.mode == "konnect" %}
+
+```bash
+helm upgrade --install kgo kong/gateway-operator -n kong-system --create-namespace --set image.tag={{ kgo_version }} \
+--set env.ENABLE_CONTROLLER_KONNECT="true" \
```

Comment:
This uses all caps for envs while https://github.com/Kong/docs.konghq.com/pull/8106/files#diff-5c601bf9f0263409a240a221ab2c362a7b04f7cc4e559bcde560c4448e9d3e59R18-R20 uses uncapitalized letters.

Let's pick 1 style to be consistent across the docs.